core/app/models/spree/promotion_rule_user.rb

```diff
@@ -0,0 +1,8 @@
+module Spree
+  class PromotionRuleUser < Spree::Base
+    self.table_name = 'spree_promotion_rules_users'
```

Comment:
We can remove this when the table is renamed. I'm looking to merge this into the `v1.0` branch, so it will stay this way for now.